### DISCUSSION TITLE
new firmware 1.7.1 available

### DISCUSSION THREAD

**Discussion by andrewoe** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#14] [General]
... uograde my EZ-1 yesterday. No problems until now... no hint or documentation from APSystems available, unfortunately...

**Comment by Moerphium** [comment]
Same for me.
I just received my Inverter this week and installed every thing today. Without checking i Updated to FW 1.7.1 and had no issues at all installing and connecting for the first time.

Thx SonnenladenGmbH for this Integration!
